tsconfig.json

```diff
@@ -36,7 +36,6 @@
       "./node_modules/@types"
     ],
     "types": [
-      "jasmine",
       "node"
```

Comment:
yeah that jasmine removal should be moved to a tsconfig-test.json that we use for any `testonly=1` compilation

Reply:
ack